src/js/components/Header/stories/Sticky.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+  Anchor,
+  Avatar,
+  Card,
+  Grid,
+  Header,
+  Nav,
+  Page,
+  PageContent,
+  Paragraph,
+} from 'grommet';
+
+export const Sticky = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Page kind="narrow">
+    <PageContent background="light-3">
+      <Header sticky="scrollup" background="light-4" pad="small">
```

Comment:
We'd likely have the background fully bleed across.

```suggestion
    <PageContent background={{color: "light-3", fill: "horizontal"}}>
      <Header sticky="scrollup" background="light-4" pad={{vertical: "small"}}>
```

Reply:
I was hoping for something more like:
![Screen Shot 2022-03-29 at 3 47 19 PM](https://user-images.githubusercontent.com/1756948/160712844-70be4b70-c7e9-45d0-bf2d-d9a6adfb671e.png)

Reply:
This will expose some challenges with PageContent + sticky Header.